app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/OBMainViewController.java

```diff
@@ -562,86 +525,87 @@ public void popViewControllerZoom (RectF zoomRect)
         nextvc.start();
     }
 
-    public void popViewControllerToBottom(boolean animate)
-    {
-        if(viewControllers.size() > 1)
-        {
+    public void popViewControllerToBottom(boolean animate) {
+        if (viewControllers.size() > 1) {
             OBSectionController topvc = viewControllers.get(viewControllers.size() - 1);
             OBSectionController bottomvc = viewControllers.get(0);
             bottomvc.viewWillAppear(false);
             transition(bottomvc, topvc, false, 0.5);
             List<OBSectionController> toRemove = new ArrayList<>();
-            for(int i=1; i<viewControllers.size(); i++)
+            for (int i = 1; i < viewControllers.size(); i++)
                 toRemove.add(viewControllers.get(i));
-            if(toRemove.size() > 0)
+            if (toRemove.size() > 0)
                 viewControllers.removeAll(toRemove);
             showButtons(bottomvc.buttonFlagsWithFatController());
             showHideButtons(bottomvc.buttonFlagsWithFatController());
             bottomvc.start();
         }
     }
 
-    public OBSectionController topController ()
-    {
+    public OBSectionController topController() {
         if (viewControllers.size() > 0)
             return viewControllers.get(viewControllers.size() - 1);
         return null;
 
     }
 
-    public void render (OBRenderer renderer)
-    {
+    public void render(OBRenderer renderer) {
         TextureShaderProgram textureShader = (TextureShaderProgram) renderer.textureProgram;
         textureShader.useProgram();
         topLeftButton.render(renderer, this, renderer.projectionMatrix);
         topRightButton.render(renderer, this, renderer.projectionMatrix);
         bottomRightButton.render(renderer, this, renderer.projectionMatrix);
         bottomLeftButton.render(renderer, this, renderer.projectionMatrix);
         //
-        if (topLabel != null)
-        {
+        if (topLabel != null) {
             topLabel.render(renderer, this, renderer.projectionMatrix);
         }
     }
 
-    public void onResume()
-    {
-        if(viewControllers != null && viewControllers.size() > 0)
-        {
-            OBSectionController controller = viewControllers.get(viewControllers.size()-1);
-            if(controller != null)
+    public void onResume() {
+        if (viewControllers != null && viewControllers.size() > 0) {
+            OBSectionController controller = viewControllers.get(viewControllers.size() - 1);
+            long endTime;
+            endTime = System.currentTimeMillis() / 1000;                                                                 // timeStamps in seconds
```

Comment:
The name endTime is deceptive. We are anyways storing the value as startTime. Shouldn't it be startTime instead?